src/install/semver.zig

```diff
@@ -19,6 +19,12 @@ pub const String = extern struct {
     /// 3. If the final bit is not set, then it's a string that is stored in an external buffer.
     bytes: [max_inline_len]u8 = [8]u8{ 0, 0, 0, 0, 0, 0, 0, 0 },
 
+    /// Create an inline string
+    pub fn from(inlinable_buffer: []const u8) String {
+        std.debug.assert(inlinable_buffer.len <= max_inline_len);
```

Comment:
Guess we'll need to review existing code to see if this assertion holds :thinking:

Reply:
This would fail the assertion, for instance:
https://github.com/oven-sh/bun/blob/9dfbf57397fc79fcff76da90640cd580517f742e/src/install/install.zig#L1516